terminal.go

```diff
@@ -644,9 +649,20 @@ func (g *LightningTerminal) start() error {
 	go func() {
 		for {
 			select {
-			case err := <-g.nonCriticalErrQueue.ChanOut():
-				log.Errorf("Received error from subsystem, "+
-					"keeping litd running: %v", err,
+			case errFunc := <-g.nonCriticalErrQueue.ChanOut():
+				serviceName, err := errFunc()
+				if serviceName == accounts.ACCOUNTS_SERVICE {
```

Comment:
I wasn't really sure how to do this in a clean way, to be able to tell which subsystem the error is sent from. I'm not a big fan of the channel returning a `func() (string, error)`, but couldn't figure out a better solution. Let me know if you have a better alternative!

I guess one alternative would let it be a `queue.ConcurrentQueue[error]` queue, and send it to the accounts service only and no other service. But that kind of defeats the purpose of having a queue in the first place.

Reply:
yeah this is why I think it would be cool to have a lit subsystem manager type thing which will handle receiving errors from the various subsystems (just accounts for now) and setting the status correctly, logging etc. Then you would not need this work around